src/app/shared/models/raid.js

```diff
@@ -6,7 +6,11 @@ export const raidSchema = new Schema({
     required: true,
   },
   description: String,
-  location: [Number],
+  location: {
+    type: [Number],
+    required: true,
+    index: '2d'
```

Comment:
Adding this type of index allows location-based queries against the lat/lon values: https://docs.mongodb.com/manual/core/2d/